src/manifest-firefox/manifest.json

```diff
@@ -10,16 +10,20 @@
   "browser_specific_settings": {
     "gecko": {
       "id": "sonarr-radarr-lidarr-autosearch@robgreen.me",
-      "strict_min_version": "58.0"
+      "strict_min_version": "113.0"
+    },
+    "gecko_android": {
+      "strict_min_version": "113.0"
     }
   },
   "icons": {
     "16": "content/assets/images/SonarrRadarrLidarr16.png",
-    "48": "content/assets/images/SonarrRadarrLidarr48.png"
+    "48": "content/assets/images/SonarrRadarrLidarr48.png",
+    "128": "content/assets/images/SonarrRadarrLidarr128.png"
   },
   "options_ui": {
     "page": "options.html",
-    "open_in_tab": true
+    "open_in_tab": false
```

Comment:
This was required for Android, I've added a link for the tab view in the popup for Firefox desktop.